Formula/nef.rb

```diff
@@ -0,0 +1,36 @@
+class Nef < Formula
+  desc "💊 steroids for Xcode Playgrounds"
+  homepage "https://github.com/bow-swift/nef"
+  url "https://github.com/bow-swift/nef/archive/0.5.1.tar.gz"
+  sha256 "0c4331c15aa73056abc20eadd2e400ef1c949c02a7ebcebb1aaf78b84deb5cc8"
+
+  depends_on :xcode => "11.0"
+
+  def install
+    build_project
+
+    bin.install "./bin/nef"
```

Comment:
Please add a simple script to install everything into place (à la `make install`). Over time, as the project can add or remove binaries, or doc, or man pages, … the formula becomes harder to maintain.

Reply:
Currently, I am working on a refactor to migrate to pure swift the whole scripts. I will simplify how formulae will install the binaries. 

For now, as we do not have docs, or man pages I prefer to keep everything in only one file (I think it keeps the formulae simply). Anyway, if in future we include together with the binaries man pages we will migrate to makefiles as you suggested us. wdut?